src/content/docs/en/reference/api-reference.mdx

```diff
@@ -989,23 +989,40 @@ The total number of pages.
 **Type:** `string`
 </p>
 
-Get the URL of the current page (useful for canonical URLs).
+Get the URL of the current page (useful for canonical URLs). If a value is set for [`base`](/en/reference/configuration-reference/#base), the URL starts with that value.
 
 ##### `page.url.prev`
 
 <p>
 **Type:** `string | undefined`
 </p>
 
-Get the URL of the previous page (will be `undefined` if on page 1). If a value is set for [`base`](/en/reference/configuration-reference/#base), prepend the base path to the URL.
+Get the URL of the previous page (will be `undefined` if on page 1). If a value is set for [`base`](/en/reference/configuration-reference/#base), the URL starts with that value.
 
 ##### `page.url.next`
 
 <p>
 **Type:** `string | undefined`
 </p>
 
-Get the URL of the next page (will be `undefined` if no more pages). If a value is set for [`base`](/en/reference/configuration-reference/#base), prepend the base path to the URL.
+Get the URL of the next page (will be `undefined` if no more pages). If a value is set for [`base`](/en/reference/configuration-reference/#base), the URL starts with that value.
```

Comment:
This was already the case in the existing docs, but taking advantage of this PR to tweak this and make it consistent with the others:

```suggestion
Get the URL of the next page (will be `undefined` if on the last page). If a value is set for [`base`](/en/reference/configuration-reference/#base), the URL starts with that value.
```